capstone/src/main/webapp/multilinevisualization.js

```diff
@@ -132,11 +136,18 @@ function createMultilineVisualization() {
       .attr('font-weight', 'bold')
       .text('# of Page Views');
 
+  // Create random colors array, one per line
+  const colors = [];
+  for (let i = 0; i < data.restaurantData.length; i++) {
+    const rand = Math.random();
+    const color = 'hsl(" + rand*360 + ",100%,50%';
```

Comment:
It looks like it's warning you about rand being unused. You seem to be making it part of a string; is that on purpose? Is it possible this string gets evaluated in a different scope, and that this is wrong? Should you be making this string be the value of rand, so it doesn't just set it to (I assume) zero in some different scope when it gets evaluated?

Reply:
Oh gosh, thanks for double-checking that! I think the quotes got all messed up when I ran make pretty. It should be fixed now with just Math.random() * 360 calculated and not in the string. Thank you!!